Source/Scene/Camera.js

```diff
@@ -3681,7 +3681,60 @@ var cartoArray = [
   new Cartographic(),
   new Cartographic(),
 ];
-function addToResult(x, y, index, camera, ellipsoid, computedHorizonQuad) {
+
+function rotateQuad(quad, numRotations) {
+  var newQuad = new Array(4);
+  for (var i = 0, j = numRotations; i < 4; i++, j = (j + 1) % 4) {
+    newQuad[j] = quad[i];
+  }
+  return newQuad;
+}
+
+function generalHeading(heading) {
+  if (heading > 45 && heading <= 135) {
+    return "east";
```

Comment:
Just a minor nit, but I don't think we usually do returns of strings like this. Maybe an enum for this function is overkill though..

Reply:
I removed the strings and just returned the `headingQuadrant` enum values